### PR TITLE
Test PR

### PR DESCRIPTION
**Pull request by orenc1**
test PR

```release-note
NONE
```

### CODE REVIEW

**Comment by ovirt-infra** [comment]
Hello contributor, thanks for submitting a PR for this project! 

I am the bot who triggers "standard-CI" builds for this project.
As a security measure, I will not run automated tests on PRs that are not from white-listed contributors.

In order to allow automated tests to run, please ask one of the project maintainers to review the code and then do one of the following:

1. Type `ci test please` on this PR to trigger automated tests for it.
2. Type `ci add to whitelist` on this PR to trigger automated tests for it and also add you to the contributor white-list so that your future PRs will be tested automatically. ( keep in mind this list might be overwritten if the job XML is refreshed, for permanent whitelisting, please follow #3 option )
3. If you are planning to contribute to more than one project, maybe it's better to ask them to add you to the project organization, so you'll be able to run tests for all the organization's projects.

File: deploy/crds/containerized-data-importer00.crd.yaml [hunk 1]
```diff
@@ -82,8 +84,10 @@ spec:
               type: string
           type: object
       type: object
-  version: v1alpha1
   versions:
-  - name: v1alpha1
+  - name: v1beta1
     served: true
     storage: true
+  - name: v1alpha1
```
Comment (nunnatsa) [inline deploy/crds/containerized-data-importer00.crd.yaml:91]: Is it working? in v1.20.0, v1alpha1 does not exists

Comment (orenc1) [reply deploy/crds/containerized-data-importer00.crd.yaml:91]: this manifest is generated by build-manifests.sh --> csv-generator

Comment (orenc1) [reply deploy/crds/containerized-data-importer00.crd.yaml:91]: this is coming from:
https://github.com/kubevirt/containerized-data-importer/blob/master/pkg/operator/resources/operator/operator.go#L301

**Comment by orenc1** [comment]
closing in favor of #709 
/close

**Comment by orenc1** [comment]
/hold
using this PR for tests

**Pull request closed without merging**